django/utils/translation/trans_real.py

```diff
@@ -195,9 +195,7 @@ def translation(language):
     Return a translation object in the default 'django' domain.
     """
     global _translations
-    if language not in _translations:
-        _translations[language] = DjangoTranslation(language)
-    return _translations[language]
+    return _translations.setdefault(language, DjangoTranslation(language))
```

Comment:
Not sure this is desired, as it will create a `DjangoTranslation` instance for each `translation` call while the previous code wasn't.

Reply:
@charettes You are right. Thanks for response~